### PR TITLE
One-command support for JAX GPU

### PR DESCRIPTION
**Pull request by digicosmos86**
After this pr is merged, one can install `hssm` with GPU support with `pip install hssm[cuda12]` (not available on cuda)

### CODE REVIEW

**Comment by AlexanderFengler** [comment]
not available on `conda` :)

File: pyproject.toml [hunk 1]
```diff
@@ -25,7 +25,7 @@ bambi = "^0.13.0"
 numpyro = "^0.15.0"
 hddm-wfpt = "^0.1.4"
 seaborn = "^0.13.2"
-jax = "^0.4.23"
+jax = { version = "^0.4.23", extras = ["cuda12"], optional = true }
```
Comment (AlexanderFengler) [inline pyproject.toml:28]: I think for now we have to cap jax roughly at <= 0.4.26, because of the config issue... setting floatX breaks on newest.

**Review by AlexanderFengler — APPROVED** [review]
high level question: 
Does this operate via a "synonym" on pypi (listed under HSSM), or do we basically end up with two packages on pypi with this?

File: pyproject.toml (hunk 1 above)
Comment (digicosmos86) [reply pyproject.toml:28]: Looks like they fixed this? I have jax==0.4.28 and this issue did not pop up for me...

**Comment by digicosmos86** [comment]
> 

Correct. conda lol

File: pyproject.toml (hunk 1 above)
Comment (digicosmos86) [reply pyproject.toml]: Fixed

**Pull request merged**